src/test/java/uk/gov/ons/ctp/integration/ratelimiterclient/RateLimiterClientEqLaunchTest.java

```diff
@@ -0,0 +1,163 @@
+package uk.gov.ons.ctp.integration.ratelimiterclient;
+
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+import uk.gov.ons.ctp.common.error.CTPException;
+import uk.gov.ons.ctp.integration.ratelimiter.model.LimitDescriptor;
+import uk.gov.ons.ctp.integration.ratelimiter.model.RateLimitRequest;
+
+@RunWith(MockitoJUnitRunner.class)
+public class RateLimiterClientEqLaunchTest extends RateLimiterClientTestBase {
+
+  @Test
+  public void shouldRejectNullDomain() {
+    CTPException exception =
+        assertThrows(
+            CTPException.class,
+            () -> {
```

Comment:
Don't think you need the brace - it's just an expression lamda

Reply:
done